### PR TITLE
adhoc/documentation

### PR DESCRIPTION
**Pull request by allison-code-dot-org**
This PR adds documentation and tests to models we plan to surface to users and moves files/ marts around to be cleaner for launch. 

I'm still doing a full rebuild on the branch to ensure the new tests pass, but committing this now for review.

### CODE REVIEW

File: dbt/models/marts/districts/_districts__models.yml [hunk 1]
```diff
@@ -0,0 +1,79 @@
+version: 2
+
+models:
```
Comment (jordan-springer) [inline dbt/models/marts/districts/_districts__models.yml:3]: FWIW I don't usu. include every column here, I always include primary and foreign keys and any important or "testable" fields.

File: dbt/models/marts/students/dim_student_courses.sql [hunk 2]
```diff
@@ -46,38 +46,21 @@ school_years as (
...
-        --else null
-        end                     as course_name   
-
-
-        ,min(ul.created_at)     as first_activity_at 
-		,max(ul.created_at)     as last_activity_at
+        ul.user_id                              as student_id,
+        sy.school_year,
+        case 
+            when cs.course_name_true in (
```
Comment (jordan-springer) [inline dbt/models/marts/students/dim_student_courses.sql:52]: a handy test here is the `accepted values` test which we could supply just the course names we want to see come through

**Review by jordan-springer — APPROVED** [review]
gucci 👍🏽

File: dbt/models/marts/districts/_districts__models.yml (hunk 1 above)
Comment (allison-code-dot-org) [reply dbt/models/marts/districts/_districts__models.yml:3]: good note, TY

**Pull request merged**